cloud/kubernetes/helm/yugabyte/values.yaml

```diff
@@ -60,6 +60,8 @@ PodManagementPolicy: Parallel
 
 enableLoadBalancer: True
 
+isMultiAz: False
```

Comment:
If this is set, at the end of the deploy you need to set the config in the master also, its an API call. See this step: https://docs.yugabyte.com/latest/deploy/public-clouds/aws/#7-configure-az-region-aware-placement

Reply:
I believe this is also done through YW during the create universe step.